### DISCUSSION TITLE
Versioning

### DISCUSSION THREAD

**Discussion by mpejcoch** [metno/MMS#22] [Ideas]
We have been discussing if we should version the ProductEvent schema.

Would adding a `EventSchemaVersion` with `<major>.<minor>` suffice?

Major changes for compatibility breaking changes, the minor mostly for adding fields. 

https://github.com/metno/MMS/issues/20

@havardf @johtoblan @vkbo

**Comment by johtoblan** [comment]
EventSchemaVersion as part of ProductEvent? Looks good to me
